src/sass/components/_forms.scss

```diff
@@ -150,3 +150,22 @@ label abbr[title] {
 .has-error .help-block {
   color: theme-color('danger');
 }
+
+.alert .error-list-item {
+  @extend .my-1;
```

Comment:
Use `margin-top: map-get($spacers, 1);` and `margin-bottom: map-get($spacers, 1);` instead. Extending should be avoided if it's not necessary.

Reply:
5da2802